src/object/srv_obj_migrate.c

```diff
@@ -637,7 +637,9 @@ migrate_pool_tls_lookup_create(struct ds_pool *pool, unsigned int version, unsig
 	arg.svc_list = (d_rank_list_t *)entry->dpe_val_ptr;
 	arg.obj_ult_cnts = tls->mpt_obj_ult_cnts;
 	arg.dkey_ult_cnts = tls->mpt_dkey_ult_cnts;
-	rc = dss_task_collective(migrate_pool_tls_create_one, &arg, 0);
+	rc = ds_pool_task_collective(pool->sp_uuid,
+				     PO_COMP_ST_NEW | PO_COMP_ST_DOWN | PO_COMP_ST_DOWNOUT,
+				     migrate_pool_tls_create_one, &arg, false);
```

Comment:
I am not 100% sure if the "PO_COMP_ST_NEW | PO_COMP_ST_DOWN | PO_COMP_ST_DOWNOUT" tgt don't need the tls. Checked a while looks fine so far.

BTW, the last "false" better to be 0.